ethereum/forge-test/query/QueryResponse.t.sol

```diff
@@ -74,15 +75,18 @@ contract TestQueryResponse is Test {
         uint8 _numPerChainResponses,
         bytes memory _perChainResponses
     ) internal pure returns (bytes memory){
-        return abi.encodePacked(
+        bytes memory queryRequest = QueryTest.buildOffChainQueryRequestBytes(
+            _queryRequestVersion,
+            _queryRequestNonce,
+            _numPerChainQueries,
+            _perChainQueries
+        );
+        return QueryTest.buildQueryResponseBytes(
             _version,
             _senderChainId,
             _signature,
             _queryRequestLen,
```

Comment:
Feels weird passing in the length here. Can we instead fetch the length from `queryRequest` we've just built?